addons/languages/songhay/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
Should be set to `false`, until i have uploaded a signed version to the Play Store. Please change

Reply:
Thanks, Menny.  I changed it to false.